src/scopes/grants/index.js

```diff
@@ -29,8 +29,14 @@ export const topicToQuery = {
     ctn: (query) => withStateCode(query),
   },
   startDate: {
-    bef: (query) => beforeStartDate(query),
-    aft: (query) => afterStartDate(query),
+    bef: (query) => activeBefore(query),
```

Comment:
Maybe this is a sign we need to refactor filters/scopes a bit, but I _could_ see a future where we want to filter grants by their start date. The way `activeBefore` works means if we tried to reuse this filter to actually filter grants by start date we will get the wrong result. I'd hate for a developer in the future to ask for `grants` with a start date before x date and get back `grants` with an end date before x.

Also we could probably use a comment on why we have a `startDate` filter that actually filters on `endDate`. Took me a while to understand what was going on, maybe my brain has yet to fully turned on today!

Reply:
I just pushed up a proposed solution, there's some comments elaborating, but basically tried to add an options object to filterToScopes that allows for different filter options to be returned and can be built upon as we go